examples/src/main/java/org/apache/ignite/example/rebalance/RebalanceExample.java

```diff
@@ -40,17 +40,32 @@
  * <ol>
  *     <li>Import the examples project into you IDE.</li>
  *     <li>
+ *         Download and prepare artifacts for run Ignite node using the CLI tool (if not done yet):<br>
+ *         {@code ignite init}
+ *     </li>
+ *     <li>
  *         Start <b>two</b> nodes using the CLI tool:<br>
  *         {@code ignite node start --config=$IGNITE_HOME/examples/config/ignite-config.json my-first-node}<br>
  *         {@code ignite node start --config=$IGNITE_HOME/examples/config/ignite-config.json my-second-node}
  *     </li>
+ *     <li>
+ *         Cluster initialization using the CLI tool (if not done yet):<br>
+ *         {@code ignite cluster init --cluster-name=ignite-cluster --node-endpoint=localhost:10300 --meta-storage-node=my-first-node}
+ *     </li>
  *     <li>Run the example in the IDE.</li>
  *     <li>
- *         When requested, start another two nodes using the CLI tool:
+ *         When requested, start another <b>two</b> nodes using the CLI tool:
  *         {@code ignite node start --config=$IGNITE_HOME/examples/config/ignite-config.json my-first-additional-node}<br>
  *         {@code ignite node start --config=$IGNITE_HOME/examples/config/ignite-config.json my-second-additional-node}
  *     </li>
  *     <li>Press {@code Enter} to resume the example.</li>
+ *     <li>
+ *         Stop <b>four</b> nodes using the CLI tool:<br>
+ *         {@code ignite node stop --config=$IGNITE_HOME/examples/config/ignite-config.json my-first-node}<br>
```

Comment:
What's going on here? Why tf would you need to pass a config file when you're stopping a node?